conda/cli/conda_argparse.py

```diff
@@ -531,6 +531,15 @@ def configure_parser_config(sub_parsers):
         default=[],
         metavar=('KEY', 'VALUE'),
     )
+    config_modifiers.add_argument(
+        "--clear",
+        nargs=1,
+        action="append",
+        help="""Clear a list key. This clears out
+    all values for the key.""",
+        default=[],
+        metavar="KEY",
+    )
```

Comment:
Let's tweak this slightly to avoid the [trailing comma that @jaimergp pointed out](https://github.com/conda/conda/pull/11627#discussion_r927447722).

```suggestion
    config_modifiers.add_argument(
        "--clear",
        action="append",
        help="Clear a list key. This clears out all values for the key.",
        metavar="KEY",
    )
```